### PR TITLE
Revised discount articles

### PR DESCRIPTION
**Pull request by justinchoi001**
1. Revised introduction
2. Revised creation doc
3. Added more screenshots

### CODE REVIEW

File: docs/commerce/2.x/en/promoting-products/creating-a-discount.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
+# Creating a Discount
+
+Discounts can be applied to a combination of products and target customers or a reduced shipping costs. Furthermore, discounts can be set as a flat sum or as a percentage.
```
Comment (jrhoun) [inline docs/commerce/2.x/en/promoting-products/creating-a-discount.md:3]: > Discounts can be applied to a combination of products and target customers or a reduced shipping costs.

Grammar issue? I can't follow this sentence.

File: docs/commerce/2.x/en/promoting-products/introduction-to-discounts.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
+# Introduction to Discounts
+
+Discounts are created in the _Control Panel_ &rarr; _Commerce_ &rarr; _Discounts_. Users can create multiple types of discounts, from reducing the price for specific products to offering free shipment. Furthermore, a discount can be applied to all buyers, a specific [Account Group](../account-management/creating-a-new-account-group.md), or to a more narrowly defined group of customers that meet certain qualifications.
```
Comment (jrhoun) [inline docs/commerce/2.x/en/promoting-products/introduction-to-discounts.md:3]: > Discounts are created in the _Control Panel_ &rarr; _Commerce_ &rarr; _Discounts_.

Grammar - this sentence is incomplete - likely missing the word "menu" at the end.

File: docs/commerce/2.x/en/promoting-products/introduction-to-discounts.md [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+
+-->
+
+## Coupon Codes
+
+Regardless of which discount type you choose, you can make a discount more exclusive by requiring a coupon code. You can generate a coupon code then share it through emails, newsletters, or ads.
+
+![Activate the Coupon Code feature and enter a coupon code for this discount type.](./introduction-to-discounts/images/02.png)
+
+To enable customers to enter the coupon code, the _Coupon Code Entry_ widget must be deployed on a store's [Checkout](../creating-store-content/commerce-storefront-pages/checkout.md) page. (The Checkout page is created automatically if you had used the [Minium accelerator](../starting-a-store/using-the-minium-accelerator-to-jump-start-your-b2b-store.md) to build the store site.) In this example, the _Coupon Code Entry_ widget is placed on the page so that customers can enter their coupon code while they review their order.
```
Comment (jrhoun) [inline docs/commerce/2.x/en/promoting-products/introduction-to-discounts.md:37]: > To enable customers to enter the coupon code,
Wordsmithing suggestion because the current wording is challenging:

In order to accept coupon codes, the _Coupon Code Entry_ widget ...

File: docs/commerce/2.x/en/promoting-products/introduction-to-discounts.md [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+
+![Place the Coupon Code Entry widget on the Checkout page.](./introduction-to-discounts/images/07.png)
+
+Once the customer has enter his coupon code to apply the discount, the discount is reflected in the updated Order Summary.
```
Comment (jrhoun) [inline docs/commerce/2.x/en/promoting-products/introduction-to-discounts.md:41]: > Once the customer **has enter** his coupon code to apply the discount,
Grammar: Mixed tense

File: docs/commerce/2.x/en/promoting-products/introduction-to-discounts.md [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+
+![Once applied, the discount is reflected in the order summary.](./introduction-to-discounts/images/08.png)
+
+## Discount Percentages
```
Comment (jrhoun) [inline docs/commerce/2.x/en/promoting-products/introduction-to-discounts.md:45]: Clarity:
> `## Discount Percentages`

Is this section about Discount Percentages, or is it about using "Levels"? Or are Levels incidental to the topic of Discount Percentages. Depending on the answer, the name of this header should change to be clearer.

File: docs/commerce/2.x/en/promoting-products/introduction-to-discounts.md [hunk 6]
```diff
@@ -0,0 +1,79 @@
...
+
+If you want to use a percentage instead of a flat rate, you can create multiple tiers (Levels). For example, you can set a base discount of 20% off as Level 1 and then offer an additional 5% off as Level 2 if your customer makes the purchase before a deadline.
+
+![Set different levels for a discount based on a percentage.](./introduction-to-discounts/images/06.png)
+
+## Expiration Date
+
+Usually, a promotion or a discount has a set duration, like a summer special or a Christmas discount. In the _Schedule_ section, you can add an expiration date.
```
Comment (jrhoun) [inline docs/commerce/2.x/en/promoting-products/introduction-to-discounts.md:53]: Simplify

> Usually, a promotion or a discount has a set duration, like a summer special or a Christmas discount. In the _Schedule_ section, you can add an expiration date.

Simplify the wording here to be more direct and concise.

Suggestion: "Discounts can be configured to have expiration dates to coincide with promotional efforts."

**Comment by jrhoun** [comment]
Hey @justinchoi001 please see my remarks - i'll leave this pull open but I'd at least like to see the grammar issues resolved before I re-consider this one. Thanks.

File: docs/commerce/2.x/en/promoting-products/creating-a-discount.md (hunk 1 above)
Comment (justinchoi001) [reply docs/commerce/2.x/en/promoting-products/creating-a-discount.md:3]: Revised.

File: docs/commerce/2.x/en/promoting-products/introduction-to-discounts.md (hunk 2 above)
Comment (justinchoi001) [reply docs/commerce/2.x/en/promoting-products/introduction-to-discounts.md:3]: revised

File: docs/commerce/2.x/en/promoting-products/introduction-to-discounts.md (hunk 3 above)
Comment (justinchoi001) [reply docs/commerce/2.x/en/promoting-products/introduction-to-discounts.md:37]: Revised

File: docs/commerce/2.x/en/promoting-products/introduction-to-discounts.md (hunk 4 above)
Comment (justinchoi001) [reply docs/commerce/2.x/en/promoting-products/introduction-to-discounts.md:41]: Revised

File: docs/commerce/2.x/en/promoting-products/introduction-to-discounts.md (hunk 6 above)
Comment (justinchoi001) [reply docs/commerce/2.x/en/promoting-products/introduction-to-discounts.md:53]: Revised

File: docs/commerce/2.x/en/promoting-products/introduction-to-discounts.md (hunk 2 above)
Comment (justinchoi001) [reply docs/commerce/2.x/en/promoting-products/introduction-to-discounts.md:3]: Revised the entire paragraph.

File: docs/commerce/2.x/en/promoting-products/introduction-to-discounts.md (hunk 3 above)
Comment (justinchoi001) [reply docs/commerce/2.x/en/promoting-products/introduction-to-discounts.md:37]: Revised

**Comment by jrhoun** [comment]
Just pushed this up. See my revisions and fixes here @justinchoi001 https://github.com/liferay/liferay-learn/commit/6e7fd120077a0cb2d039379bdd3068bd5db1420c.

Thank you!
